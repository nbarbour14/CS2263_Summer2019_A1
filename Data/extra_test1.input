u 10
u 20
o
u 30
e
u 20
x
