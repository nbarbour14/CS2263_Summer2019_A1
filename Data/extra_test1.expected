10
20
20
30
30
20
Successfully executed 6 instructions
|_ 10 _|
|_ 30 _|
|_ 20 _|
3 elements
